README.md

```diff
@@ -0,0 +1,66 @@
+# ✨ FUN Config ✨
+
+This repository is the place where we store various configuration files shared
+among the FUN development team.
+
+## GitHub Labels
+
+Inspired by TailorDev's work on [communication with GitHub
+issues](https://tailordev.fr/blog/2016/09/27/communication-with-github-issues-1/),
+we decided to define our own set of labels that will be used in all our GitHub
+repositories. In this section, we will document how to install and use the
+[`labels`](https://github.com/popomore/github-labels) tool to automate labels
```

Comment:
At this point of my reading I was yet not sure to understand what you were talking about. I propose:
"...we decided to define our own set of labels that will be used in all our GitHub repositories to improve and normalize work on issues and PR for internal and external contributors."